<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Хэш таблица (линейное зондирование)</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../Kittens_attack_2!/sitting-cat.ico</normaloff>../../../../Kittens_attack_2!/sitting-cat.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabInsertionInfo">
       <attribute name="title">
        <string>Вставка</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTextEdit" name="textInsertionInfo"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabRemovingInfo">
       <attribute name="title">
        <string>Удаление</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTextEdit" name="textRemovingInfo"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
